redaxo/src/core/lib/console/setup/run.php

```diff
@@ -212,6 +335,58 @@ static function ($value) {
             $config['db'][1]['password'] = $dbPassword;
             $config['db'][1]['name'] = $dbName;
 
+            // CLI SSL configuration (existing non-interactive logic)
+            if ($input->hasParameterOption('--db-ssl-ca') || $input->getOption('db-ssl-key') || $input->getOption('db-ssl-cert') || $input->getOption('db-ssl-verify-server-cert')) {
+                // Ensure db config array exists
+                if (!isset($config['db']) || !is_array($config['db'])) {
+                    $config['db'] = [];
+                }
+                if (!isset($config['db'][1]) || !is_array($config['db'][1])) {
+                    $config['db'][1] = [];
+                }
```

Comment:
beide if sind unnötig, `$config['db'][1]` wird ja schon in zeile 333-336 gesetzt.

Reply:
Die doppelte $dbConfig['ssl'] Zuweisung wurde bereinigt, nur eine @var Annotation für Psalm wurde beibehalten (für sichere Array-Extraktion bei $dbConfigArray[1])

Reply:
Gerne. Du bist hier mehr zu Hause :-) ... hoffe es ist nützlich